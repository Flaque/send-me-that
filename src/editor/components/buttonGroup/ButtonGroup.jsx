import React from 'react';

class ButtonGroup extends React.Component {
  render() {
    return (
      <div className="buttonGroup">
        <button id="run" className="primary" onClick={this.props.onRun}> Run </button>

        {this.props.showInsert &&
          <button id="insert" onClick={this.props.onInsert}> Insert </button>
        }
      </div>
    )
  }
}

export default ButtonGroup;
