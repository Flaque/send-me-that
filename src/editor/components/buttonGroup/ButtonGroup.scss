.buttonGroup {
  margin-top: 10px;

  button {
    margin-right: 10px;
  }

  #run {
    float: left;
  }

  #insert {
    float: left;
  }
}
