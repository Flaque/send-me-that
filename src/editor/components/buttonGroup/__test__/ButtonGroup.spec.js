import React from 'react';
import ButtonGroup from '../ButtonGroup.jsx';
import renderer from 'react-test-renderer';

test('Test that ButtonGroup is still the same', () => {
  const tree = renderer.create(
    <ButtonGroup />
  ).toJSON();
  expect(tree).toMatchSnapshot();
})
