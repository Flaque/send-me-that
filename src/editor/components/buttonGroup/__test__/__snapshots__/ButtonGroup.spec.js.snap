exports[`test Test that ButtonGroup is still the same 1`] = `
<div
  className="buttonGroup">
  <button
    className="primary"
    id="run"
    onClick={undefined}>
     Run 
  </button>
</div>
`;
